components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/annotations/metadata_general.html

```diff
@@ -105,6 +105,8 @@
                     $("#link_info_popup").show();
                 {% if webclient_path %}
```

Comment:
@will-moore what was the logic behind https://github.com/manics/openmicroscopy/commit/a9d87d9d8bf1a3ad04e89a7137a55823f9726ce5#diff-5ecd969a9790526e02d15fcb673d6ca5R119 vs https://github.com/manics/openmicroscopy/commit/4eebe363cd36bdc0e358162c12dd057cdfbb7f19#diff-5ecd969a9790526e02d15fcb673d6ca5R116 At the moment we have both in place and we are wondering when `else` is used?